src/extensions/CanvasWidget/CanvasWidget.tsx

```diff
@@ -100,10 +101,9 @@ const CanvasWidget = <T extends BaseLayerProps>(props: Props<T>) => {
         // or when the layers (prop) has changed (or if preventDefaultWheel has changed)
         // we set the canvas elements on the layers and schedule repaints
         if (!divElement) return
-        if (!layers) return
         layers.forEach((L, i) => {
             const canvasElement = divElement.children[i]
-            if (canvasElement) {
+            if ((canvasElement) && (L)) {
```

Comment:
The other change is that individual layers passed in to CanvasWidget can be temporarily null, where as the layers prop is always a list now and is not allowed to be null. Previously the layers could be null, but if not null, the individual layers would need to be null. So this is why we now need to check if L !== null throughout.